### PR TITLE
chore: update to ipfs 0.44.0

### PR DESCRIPTION
**Pull request by oed**
This was fairly straight forward, just a bunch of incompatibilities to crush though. 

Right now all tests should be passing. However I have not teseted anything manually yet.

Also note that this disables 3id-connect for now.

### CODE REVIEW

**Comment by oed** [comment]
Ok, need to figure out why tests are not passing on circleci.

**Comment by oed** [comment]
Alright, tests should be stable now 🙌

**Comment by oed** [comment]
IdWallet integration test is still randomly failing -.-
Looking into way to improve it.

**Comment by zachferland** [comment]
With  '/dns4/p2p.3box.io/tcp/9091/wss/p2p-webrtc-star/' enabled, usage in safari blocked, 'Unhandled Promise Rejection: Error: no WebRTC support' thrown on create. desktop safari should have it though, but may not be available in ios app browsers at all. 

Then anywhere getting websocket connection dropped 

```WebSocket connection to 'wss://ipfs.3box.io/p2p/QmZvxEpiVNjmNbEKyQGvFzAY1BwmGuuvdUTmcTstQPhyVC' failed: WebSocket is closed before the connection is established.

Unhandled Promise Rejection: AggregateError:```

Did we deploy the /p2p change yet for prod? cc @oed @simonovic86

**Comment by zachferland** [comment]
Working well in chrome for me now against prod. Can connect in safari with not webrtc service, but still doesnt work there. Errors look same as here- https://github.com/3box/3id-connect/issues/49 and maybe worse than before, but not clear

**Comment by oed** [comment]
Made an update here to not break if there is no webrtc support. Now we just simply log a warning in the console if webrtc is missing.

@zachferland where you still seeing issues with webrtc disabled?

**Comment by zachferland** [comment]
Yeah, this  twice, not seen before 

[Error] Unhandled Promise Rejection: TypeError: timeoutController.clear is not a function. (In 'timeoutController.clear()', 'timeoutController.clear' is undefined)
	(anonymous function) (index.js:160)
	asyncFunctionResume
	(anonymous function)
	promiseReactionJobWithoutPromise
	promiseReactionJob

**Comment by zachferland** [comment]
also if needed, using this branch with web3modal to test in safari - https://github.com/3box/3box-js/pull/832

**Comment by zachferland** [comment]
also disable option doesnt work, i just guess also just check if address available

**Comment by zachferland** [comment]
Latest error I saw - Unhandled Promise Rejection: TypeError: undefined is not an object (evaluating 'ipfs.libp2p.transportManager.listen') 

last errors were before you pushed up some more stuff

**Comment by oed** [comment]
Ehm, make sure to run `npm ci` before you run the example @zachferland!

**Comment by zachferland** [comment]
ok, yeah looks back to original errors i posted above

**Comment by zachferland** [comment]
error explained by - https://github.com/libp2p/js-libp2p/issues/568

File: .circleci/config.yml [hunk 1]
```diff
@@ -18,14 +18,14 @@ jobs:
...
 
-      - run:
-          name: test
-          command: npm test && codecov
-
       - run:
           name: lint
           command: npm run lint
 
+      - run:
```
Comment (zachferland) [inline .circleci/config.yml:25]: 👍

**Pull request merged**